src/blocktree_processor.rs

```diff
@@ -132,11 +151,26 @@ fn process_block(
     leader_scheduler: &Arc<RwLock<LeaderScheduler>>,
 ) -> Result<()> {
     for entry in entries {
-        process_entry(bank, entry)?;
+        let (res, fee) = process_entry(bank, entry);
+        let num_ticks = bank.tick_height();
+        let slot_height = leader_scheduler
```

Comment:
Before merging this, can you add `get_leader_for_tick` method to `LeaderScheduler` and replace all the instances of this exact pattern?

Reply:
sure